docs/user-guide/join-a-jitsi-meeting.md

```diff
@@ -4,6 +4,11 @@ title: Join a Jitsi Meeting
 sidebar_label: Join a Jitsi Meeting
 ---
 
-Welcome to the user guide!
 
-Check back soon!
+1. When visiting a Jitsi Meeting link, your browser may first ask you to grant microphone and/or camera access.
```

Comment:
As this is only one method, you might add a headline like:
`Join by using a hyperlink`

Also you might describe the URL method a bit more:
- people should get the url from a trusted person over a trusted channel
- `Copy the url into the browser and click enter` (this might seems obvious, but this guide is also intended for people who don't really have a clue)